src/katgpucbf/fgpu/recv.py

```diff
@@ -210,9 +212,56 @@ def make_streams(
     return streams
 
 
+def make_sensors(sensor_timeout: float) -> aiokatcp.SensorSet:
+    """Create the sensors needed to hold receiver statistics.
+
+    Parameters
+    ----------
+    sensor_timeout
+        Time (in seconds) without updates before sensors for received data go
+        into error and sensors for missing data become nominal.
+    """
+    sensors = aiokatcp.SensorSet()
+    for pol in range(N_POLS):
+        timestamp_sensors: list[aiokatcp.Sensor] = [
+            aiokatcp.Sensor(
+                int,
+                f"input{pol}-rx-timestamp",
+                "The timestamp (in samples) of the last chunk of data received from the digitiser",
+                default=-1,
+                initial_status=aiokatcp.Sensor.Status.ERROR,
+            ),
+            aiokatcp.Sensor(
+                float,
+                f"input{pol}-rx-unixtime",
+                "The timestamp (in UNIX time) of the last chunk of data received from the digitiser",
+                default=-1.0,
+                initial_status=aiokatcp.Sensor.Status.ERROR,
+            ),
+        ]
+        for sensor in timestamp_sensors:
+            TimeoutSensorStatusObserver(sensor, sensor_timeout, aiokatcp.Sensor.Status.ERROR)
+            sensors.add(sensor)
+        missing_sensors: list[aiokatcp.Sensor] = [
```

Comment:
I see you've added empty lines in fgpu's `Engine::_populate_sensors` to separate the concerns
* Might I request the same here after you've created the 'normal' `rx-time` sensors (please)?